### PR TITLE
Feature/optimize fec loading

### PR DESCRIPTION
**Pull request by anthonygarvan**
This includes regulations-parser and appropriate loading scripts to streamline loading FEC regs into eregs. Also detailed instructions in README.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ $ python manage.py runserver
 ### Loading FEC's regulations
 
 You will need access to FEC's org in cloud.gov for this.
-Make sure you have run `pip install -r requirements_dev.txt`.
+Make sure you have run `pip install -r requirements.txt && pip install -r requirements_dev.txt`.
```
Comment (vrajmohan) [inline README.md]: Wouldn't it be better to just _include_ `requirements.txt` in `requirements_dev.txt`?

File: README.md [hunk 2]
```diff
@@ -58,17 +58,21 @@ load_regs/fec_reg_parts.txt.
...
 $ cf create-service aws-rds shared-psql fec-eregs-db
+$ cf bind-service eregs fec-eregs-db
 $ cf restage eregs
 ```
 
 Now you can load the regs with:
 
 ```bash
-$ python load_regs/load_fec_regs.py dev $HTTP_AUTH_USER $HTTP_AUTH_PASSWORD
+$ python load_regs/load_fec_regs.py [env] $HTTP_AUTH_USER $HTTP_AUTH_PASSWORD
```
Comment (vrajmohan) [inline README.md]: Minor suggestion - call the placeholder `space` as that's what they are called in Cloud Foundry.

File: README.md [hunk 3]
```diff
@@ -58,17 +58,21 @@ load_regs/fec_reg_parts.txt.
 If you are loading regs for a new year, you will need to reset the database. To do that, run:
 ```bash
 $ cf unbind-service eregs fec-eregs-db
+$ cf service-keys fec-eregs-db
```
Comment (vrajmohan) [inline README.md]: Service keys are new to me. Do we need them at all, and if we do, where/how are we creating them?

File: README.md [hunk 4]
```diff
@@ -35,7 +35,53 @@ $ python manage.py compile_frontend
 $ python manage.py runserver
 ```
 
-### Data
+### Loading FEC's regulations
+
+You will need access to FEC's org in cloud.gov for this.
+Make sure you have run `pip install -r requirements_dev.txt`.
+
+In the environment you with to update regulations, first run:
```
Comment (vrajmohan) [inline README.md]: Typo?

**Comment by pkfec** [comment]
@vrajmohan 
i am attempting to rebase and merge this PR into develop branch.

**Pull request merged**